Automattic/WP-Job-Manager
# Improve settings descriptions

Fixes #2491, #2492

Implemented with contributions from @masteradhoc 

### Changes Proposed in this Pull Request

* Updates the descriptions of the 'Filled Listings' and 'Expired Listings' settings.

### Testing Instructions

* Visit the settings page and make sure that the descriptions are correct.


----

| Plugin build for 115b58a65d675fa42efa9642ca4bf7c92036cf84 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/11/wp-job-manager-zip-2639-115b58a6.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/11/2639-115b58a6)             |
